src/Form/OgMembershipForm.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
@jludwig 
```
src/Form/OgMembershipForm.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
----------------------------------------------------------------------
 42 | WARNING | t() calls should be avoided in classes, use
    |         | dependency injection and $this->t() instead
 50 | WARNING | \Drupal calls should be avoided in classes, use
    |         | dependency injection instead
----------------------------------------------------------------------
```
To be checked if it makes sense to follow these instructions from coder.

Reply:
Both issues seem to have been resolved.